aeon/dj_pipeline/acquisition.py

```diff
@@ -612,54 +612,62 @@ class Message(dj.Part):
     def make(self, key):
         chunk_start, chunk_end = (Chunk & key).fetch1("chunk_start", "chunk_end")
 
-        raw_data_dir = Experiment.get_data_directory(key)
+        self.insert1(key)
 
+        # Populate the part table
+        raw_data_dir = Experiment.get_data_directory(key)
         device = getattr(
             _device_schema_mapping[key["experiment_name"]], "ExperimentalMetadata"
         )
-        log_messages = io_api.load(
-            root=raw_data_dir.as_posix(),
-            reader=device.MessageLog,
-            start=pd.Timestamp(chunk_start),
-            end=pd.Timestamp(chunk_end),
-        )
-        state_messages = io_api.load(
-            root=raw_data_dir.as_posix(),
-            reader=device.EnvironmentState,
-            start=pd.Timestamp(chunk_start),
-            end=pd.Timestamp(chunk_end),
-        )
 
-        self.insert1(key)
-        self.Message.insert(
-            (
-                {
-                    **key,
-                    "message_time": r.name,
-                    "message": r.message,
-                    "message_type": r.type,
-                }
-                for _, r in log_messages.iterrows()
-            ),
-            skip_duplicates=True,
-        )
-        self.Message.insert(
-            (
-                {
-                    **key,
-                    "message_time": r.name,
-                    "message": r.state,
-                    "message_type": "EnvironmentState",
-                }
-                for _, r in state_messages.iterrows()
-            ),
-            skip_duplicates=True,
-        )
+        try:
+            log_messages: pd.DataFrame = io_api.load(
+                root=raw_data_dir.as_posix(),
+                reader=device.MessageLog,
+                start=pd.Timestamp(chunk_start),
+                end=pd.Timestamp(chunk_end),
+            )
+
+            self.Message.insert(
+                (
+                    {
+                        **key,
+                        "message_time": r.name,
+                        "message": r.message,
+                        "message_type": r.type,
+                    }
+                    for _, r in log_messages.iterrows()
+                ),
+                skip_duplicates=True,
+            )
+        except:
+            print("Can't read from device.MessageLog")
```

Comment:
Let's catch specific error type(s) (e.g. `except ValueError, KeyError` something like that) - instead of an open ended `except`. 
Also, `logger.info()` or `logger.warning` is better than `print` if applicable

Reply:
yes, that should be better.